component/src/test/java/org/wso2/extension/siddhi/io/jms/sink/JMSSinkTestCase.java

```diff
@@ -78,6 +78,7 @@ public void jmsTopicPublishTest1() throws InterruptedException {
             //init
             Thread listenerThread = new Thread(client);
             listenerThread.start();
+            Thread.sleep(1000);
 
```

Comment:
Shall we remove the unnecessary spaces?

Reply:
Fixed with commit [#3fb99a6822b8f5bb32d97ebdc4b7495aa01a0dfc](https://github.com/wso2-extensions/siddhi-io-jms/pull/25/commits/3fb99a6822b8f5bb32d97ebdc4b7495aa01a0dfc)